src/wp-content/themes/twentytwentyone/assets/sass/02-tools/mixins.scss

```diff
@@ -20,32 +20,61 @@
 // Button style
 // - Applies button styles to blocks and elements that share them.
 @mixin button-style() {
-	line-height: var(--button--line-height);
-	color: var(--button--color-text);
+	border: var(--button--border-width) solid transparent;
+	border-radius: var(--button--border-radius);
 	cursor: pointer;
+	display: block;
 	font-weight: var(--button--font-weight);
 	font-family: var(--button--font-family);
 	font-size: var(--button--font-size);
-	background-color: var(--button--color-background);
-	border-radius: var(--button--border-radius);
-	border: var(--button--border-width) solid var(--button--color-background);
-	text-decoration: none;
+	font-weight: bolder;
```

Comment:
Why are there two font weights? Can we set `var(--button--font-weight);` to bolder and still make the button blocks bold text setting work?

Reply:
Ahh, yeah thats a mistake. Will remove the duplicate one shortly. 

> Can we set var(--button--font-weight); to bolder and still make the button blocks bold text setting work?

Yes, will do :-)